impeller/entity/contents/content_context.cc

```diff
@@ -604,4 +606,68 @@ void ContentContext::FlushCommandBuffers() const {
   GetContext()->GetCommandQueue()->Submit(buffers);
 }
 
+void ContentContext::InitializeCommonlyUsedShadersIfNeeded() const {
+  // Initialize commonly used shaders that aren't defaults.
+  auto options = ContentContextOptions{
+      .sample_count = SampleCount::kCount4,
+      .color_attachment_pixel_format =
+          context_->GetCapabilities()->GetDefaultColorFormat()};
+
+  for (const auto mode : {BlendMode::kSource, BlendMode::kSourceOver}) {
```

Comment:
From the proceedings of the last go/impeller-weekly, my understanding was that PSO variant construction time did not show up in traces yet. What gives for the need for this?

Reply:
The two traces posted are also unrelated to PSO variant construction and the linked bug makes no mention of PSO variants being slow to initialize. This bit seems unrelated to the linked issue or any problem we've discussed in the past.